docker-compose.yml

```diff
@@ -128,8 +128,6 @@ services:
     sysctls:
       - net.ipv6.conf.all.disable_ipv6=0
     depends_on:
-      gateway:
```

Comment:
Removing this allows us to write integration tests that test start and restart behavior of various services.